src/app/pages/component-viewer/component-viewer.ts

```diff
@@ -76,10 +76,10 @@ export class ComponentOverview implements OnInit {
 
   ngOnInit() {
     // 100ms timeout is used to allow the page to settle before moving focus for screen readers.
-    setTimeout(() => this.focusTarget.nativeElement.focus(), 100);
+    setTimeout(() => this.focusTarget.nativeElement.focus({preventScroll: true}), 100);
   }
 
-  onContentLoaded() {
+  onContentRendered() {
```

Comment:
Could we call this `scrollToSelectedContentSection"? In general it's better to name functions for what they do rather than how they're called.

Reply:
Definitely. I've just tried to follow the naming conventions that existed before. Changed now.